### PR TITLE
docs: add documentation override

### PR DESCRIPTION
**Pull request by JoanFM**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200479679796904/1200536087441892) by [Unito](https://www.unito.io)

### CODE REVIEW

File: .github/2.0/cookbooks/Hubble.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Table of Contents
   - [3. Use in Jina Flow](#3-use-in-jina-flow)
     - [3.2 using docker images](#32-using-docker-images)
     - [3.2 using source codes](#32-using-source-codes)
+    - [3.2 override default __init__ parameters](#32-override-parameters)
```
Comment (hanxiao) [inline .github/2.0/cookbooks/Hubble.md:14]: TOC in all markdown are taken care by the dev bot, no need to add this manually

File: .github/2.0/cookbooks/Hubble.md [hunk 2]
```diff
@@ -107,4 +108,15 @@ from jina import Flow
 f = Flow().add(uses='jinahub://<UUID8/Alias>[:<SECRET>]')
 ```
 
+### 3.2 override default parameters
```
Comment (hanxiao) [inline .github/2.0/cookbooks/Hubble.md:111]:
```suggestion
### 3.2 Override Default Parameters
```

File: .github/2.0/cookbooks/Hubble.md (hunk 1 above)
Comment (JoanFM) [reply .github/2.0/cookbooks/Hubble.md:14]: ok

Comment (JoanFM) [inline .github/2.0/cookbooks/Hubble.md:14]:
```suggestion
```

**Pull request merged**